pim-tool-back/src/main/java/vn/elca/training/model/entity/Project.java

```diff
@@ -1,88 +1,46 @@
 package vn.elca.training.model.entity;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.OneToMany;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
 import java.time.LocalDate;
-import java.util.HashSet;
-import java.util.Set;
 
 /**
- * @author vlp
+ * @author thomas.dang
  */
 @Entity
+@Getter
+@Setter
+@Table(name = "PROJECT")
 public class Project {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID", nullable = false)
     private Long id;
 
-    @Column(nullable = false)
-    private String name;
-
-    @Column
-    private LocalDate finishingDate;
-
-    @Column
-    private String customer;
-
-    @OneToMany(mappedBy = "project", fetch = FetchType.LAZY)
-    private Set<Task> tasks = new HashSet<>();
-
-    public Project() {
-    }
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "GROUP_ID", nullable = false)
+    private Group group;
 
-    public Project(String name, LocalDate finishingDate) {
-        this.name = name;
-        this.finishingDate = finishingDate;
-    }
+    @Column(name = "PROJECT_NUMBER", nullable = false)
+    private Integer projectNumber;
 
-    public Project(Long id, String name, LocalDate finishingDate) {
-        this.id = id;
-        this.name = name;
-        this.finishingDate = finishingDate;
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public LocalDate getFinishingDate() {
-        return finishingDate;
-    }
+    @Column(name = "NAME", nullable = false, length = 50)
+    private String name;
 
-    public void setFinishingDate(LocalDate finishingDate) {
-        this.finishingDate = finishingDate;
-    }
+    @Column(name = "CUSTOMER", nullable = false, length = 50)
+    private String customer;
 
-    public String getCustomer() {
-        return customer;
-    }
+    @Column(name = "STATUS", nullable = false, length = 3)
+    private String status;
```

Comment:
Hint: which data type in Java that forces value to be in one of the declared constant?

Reply:
use Enum similar to Role case in exercise week 2, 3?